docs/CONTRIBUTING.md

```diff
@@ -47,6 +62,22 @@ Run `make` to build `skyeye`.
 
 Anyhwere this guide mentions `skyeye.exe`, remove `.exe` - just run `skyeye`.
 
+### MACOS
```

Comment:
Let's put a note here marking this as "experimental" since I don't have an Apple Silicon device to do regression testing. It _should_ be fine as long as I avoid any new CGO dependencies 🤞🏼 

Hopefully I get one not too far in the future, my professional machine is very long in the tooth haha

Reply:
Hopefully you will get your device soon. It is so much better!